source/unsafe-messenger.js

```diff
@@ -73,6 +75,35 @@ export default function unsafeMessenger() {
 			editor.blur();
 		});
 	}
+
+	function monacoEditor(target) {
+		const editor = monaco.editor.getModels()[0];
+		let currentValue = editor.getValue();
+		sendBack(target, currentValue);
+
+		editor.onDidChangeContent((e) => {
+			// Check if the change was made by the user
```

Comment:
This comment just rephrases the code, it's not adding extra information, can be dropped